seva-launcher/seva-compose.go

```diff
@@ -45,11 +45,17 @@ func start_app(command WebSocketCommand) WebSocketCommand {
 	return command
 }
 
-func web_proxy(command WebSocketCommand) WebSocketCommand {
+func save_settings(command WebSocketCommand) WebSocketCommand {
 	log.Println("I'm in")
 	
 	// Add "sysconfig/docker file" updation code here
-    
+    err := ioutil.WriteFile("/etc/sysconfig/docker", []byte(command.Command), 0666)
+    if err != nil {
+        log.Fatal(err)
+    }
+
+	command.Response = append(command.Response, strings.Split("write-done", "\n")...)
```

Comment:
Don't use `string.Split` here. If you have a oneline response then just append the one line response.
```go
command.Response = append(command.Response, "write-done")
```